engine/class_modules/monk/sc_monk.cpp

```diff
@@ -171,6 +173,15 @@ void monk_action_t<Base>::apply_buff_effects()
   parse_effects( p()->buff.press_the_advantage );
 
   // Conduit of the Celestials
+  parse_effects( p()->buff.flight_of_the_red_crane );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr );
+  parse_effects( p()->buff.heart_of_the_jade_serpent_cdr_celestial );
+  parse_effects( p()->buff.inner_compass_crane_stance );
+  parse_effects( p()->buff.inner_compass_ox_stance );
+  parse_effects( p()->buff.inner_compass_serpent_stance );
+  parse_effects( p()->buff.inner_compass_tiger_stance );
+  parse_effects( p()->buff.jade_sanctuary );
```

Comment:
~~Jade Sanctuary doesn't modify actions.~~
~~Belongs in monk_t::parse_player_effects()~~

Jade Sanctuary only modifies Celestial Conduit, and is already parsed there.

Note: Effect NYI iirc, on my todo. Fine to move Jade Sanctuary, it'll work when I get that done.